### PR TITLE
Add story standards

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/PSD-212

### CODE REVIEW

File: docs/backlog-management.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Backlog management
+All FFC teams follow an agile delivery approach.  Teams should use whichever agile methodology best fits their delivery, for example Scrum, Kanban etc.
+
+If teams are unclear which methodology to adopt, they should start with vanilla Scrum and iterate as appropriate.  Teams looking for help getting started with Scrum can follow this [training series](http://scrumtrainingseries.com).
```
Comment (Matthew-Collins-Defra) [inline docs/backlog-management.md:6]: Might be worth adding a link to the GDS Agile delivery section of the service manual for teams that are new to agile.
https://www.gov.uk/service-manual/agile-delivery

Comment (johnwatson484) [reply docs/backlog-management.md:6]: Good idea

Comment (johnwatson484) [reply docs/backlog-management.md:6]: Link to guidance added.

Comment (Matthew-Collins-Defra) [reply docs/backlog-management.md:6]: Looks good to me.

**Review by Matthew-Collins-Defra — APPROVED** [review]
Looks good, nice summary of the relevant information.

**Pull request merged**